components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/internal/handler/UserOperationEventHookHandler.java

```diff
@@ -233,4 +239,21 @@ public void handleEvent(Event event) throws IdentityEventException {
             log.debug("Error while retrieving event publisher configuration for tenant.", e);
         }
     }
+
+    /**
+     * Where user is created regardless of confirmed account or not.
+     *
+     * @param eventName
+     * @return
+     */
+    private boolean isUserCreatedFlow(String eventName) {
+
+       /*
+        All POST_ADD_USER events will result in a userCreated event payload.
+        Since user creation does not imply successful registration,
+        this check is valid and does not cause any issues.
+         */
+        return (IdentityEventConstants.Event.POST_ADD_USER.equals(eventName) ||
+                IdentityEventConstants.Event.USER_REGISTRATION_SUCCESS.equals(eventName));
```

Comment:
Is user registration success check necessary ? Won't it fire two events. In JIT and in any user add POST_ADD_USER gets triggered right

Reply:
FIxed 6ba5e8620a2e02edc12db24e1d42babbb6a7e56c